Automattic/woocommerce-payments
# Added ENUM class for currency codes

Fixes #7887 

#### Changes proposed in this Pull Request
Created an ENUM class for defining currency constants.









#### Testing instructions





- Please do a basic smoke test
- Please check if I missed modifying something
- Make sure tests are passing

-------------------


**Post merge**
